fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersioning.java

```diff
@@ -243,4 +244,16 @@ public Response delete() {
     protected String externalPath() {
         return externalPath;
     }
+
+    @Override
+    protected FedoraResource resource() {
```

Comment:
If we POST on `/fcr:versions` when the head is a tombstone, is that ok? It seems like before it would have been returning a 410 but this would now be returning a 201.

I haven't actually tried this yet though (or know what behavior we should have), just something I was thinking about when looking at this file.